exec/java-exec/src/main/java/org/apache/drill/exec/store/httpd/HttpdLogRecord.java

```diff
@@ -109,8 +117,36 @@ public void set(final String field, final Long value) {
       if (w != null) {
         LOG.trace("Parsed field: {}, as long: {}", field, value);
         w.writeBigInt(value);
+      } else {
+        LOG.warn("No 'long' writer found for field: {}", field);
       }
-      else {
+    }
+  }
+
+  /**
+   * This method is referenced and called via reflection. This is added as a parsing target for the parser. It will get
+   * called when the value of a log field is a timesstamp data type.
+   *
+   * @param field name of field
+   * @param value value of field
+   */
+  @SuppressWarnings("unused")
+  public void setTimestamp(final String field, final String value) {
```

Comment:
Avoid using `final` unnecessarily.

Reply:
Removed this and other finals in method declarations.